live/src/js/table/Cell.js

```diff
@@ -306,13 +306,19 @@ class Cell extends React.Component {
 	};
 
 	indexCurrentData(nextData) {
-		feed.indexData({
+		const data = {
 			type: this.props._type,
 			id: this.props._id,
 			body: {
 				[this.props.columnName]: nextData
 			}
-		}, 'updateCell', res => this.handleErrorMsg(res));
+		};
+
+		if (this.props._mapping && this.props._mapping._routing && this.props._mapping._routing.required) {
```

Comment:
@bstst Here, we should also add a check for the presence of `this.props._mapping._parent` to verify that the document in the type being indexed indeed has a parent mapping.